default.nix

```diff
@@ -172,7 +181,17 @@ nightlyToolchains.${v} // rec {
       ];
     doCheck = false;
     CARGO_INCREMENTAL = 0;
-    RUST_ANALYZER_REV = rust-analyzer-rev;
+
+    # See rust-analyzer's crates/rust-analyzer/build.rs
```

Comment:
Would you mind changing this to a static link? If the file ever changes and causes something to break it will make it a bit easier to tell what changed.

Reply:
Done!